### PR TITLE
Build for other arches

### PR DESCRIPTION
**Pull request by xhochy**
Fixes #25 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/0001-Check-before-removing-march-native.patch
Comment (xhochy) [inline recipe/0001-Check-before-removing-march-native.patch:1]: Submitted upstream: https://github.com/nmslib/hnswlib/pull/476

**Comment by xhochy** [comment]
@conda-forge/hnswlib This is ready to review.

File: recipe/0001-Check-before-removing-march-native.patch
Comment (yihming) [reply recipe/0001-Check-before-removing-march-native.patch:1]: @xhochy Thank you for your help on resolving this cross-platform issue. 

I just have one question: If your PR were merged in hnswlib repo and included in its releases, would your modifications here conflict with them?

**Comment by yihming** [comment]
@xhochy Thank you for your help on resolving this cross-platform issue.

I just have one question: If your PR were merged into hnswlib repo and included in its releases, would your modifications here conflict with them?

**Comment by xhochy** [comment]
> @xhochy Thank you for your help on resolving this cross-platform issue.
> 
> 
> 
> I just have one question: If your PR were merged into hnswlib repo and included in its releases, would your modifications here conflict with them?

In a way: yes because both modifications are the same. Once the upstream PR has been merged, you would have to remove the patch from the recipe again.

**Comment by xhochy** [comment]
Can this be merged though?

**Comment by yihming** [comment]
Thanks for the explanation. I'll go ahead merging it. Thank you for your contribution!

**Review by yihming — APPROVED** [review]

**Pull request merged**